src/sas/qtgui/MainWindow/GuiManager.py

```diff
@@ -352,7 +353,11 @@ def showHelp(self, url):
         """
         Open a local url in the default browser
         """
-        GuiUtils.showHelp(url)
+        try:
+            # Help window shows itself
+            self.helpWindow = DocViewWindow(parent=self, source=url)
```

Comment:
The window should be created in the GuiManager init so a new window is not generated every time the method is called. This method should only need to update the url and show the window if it's hidden. Closing the window should then hide it instead of destroying it.

Reply:
Would this not prevent one from having more than one help window open at once?

If so, that would be very bad IMO. I often open several (all browser windows currently) to different pages so I can easily go back and forth. It would be monumentally irritating if I had to close and open and close and open back and forth. The only thing we want to avoid is opening the **same** window multiple times.

If it does not prevent the above use case then this sounds like a reasonable idea.